go/config/BUILD.bazel

```diff
@@ -74,3 +74,9 @@ filegroup(
     srcs = glob(["**"]),
     visibility = ["//visibility:public"],
 )
+
+string_flag(
```

Comment:
Have you tested the case when you pass multiple flags? `gc_goopts` and `gc_linkopts` are similar enough that I don't want to make one of them `string_list_flag`, the other `string_flag`. People reading the code in the future will be confused.

Reply:
Because of the call to `_quote_opts`, I suspect you won't be able to pass multiple gc flags if you make `gc_goopts` as `string_flag`. Can you verify?

https://github.com/bazelbuild/rules_go/blob/3d19fb53089ddbaf3732c4083cd1a9e2066f08b1/go/private/actions/compilepkg.bzl#L132

Reply:
I did initially - not since appending to mode though, I thought _quote_opts would mess with things too but gc_flags is a list that includes race and msan conditions
iirc it shows up as -gcflags '--first=1 --second=2'
going to try to see output now

Reply:
I can switch it to string_list flag it does contain escaped quotes in this case
`bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64 -src tests/core/go_binary/hello.go -embedroot '' -embedroot bazel-out/k8-fastbuild/bin -embedlookupdir tests/core/go_binary -importpath hello_nopie_bin -p main -package_list bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/packages.txt -o bazel-out/k8-fastbuild/bin/tests/core/go_binary/hello_nopie_bin.a -x bazel-out/k8-fastbuild/bin/tests/core/go_binary/hello_nopie_bin.x -gcflags ''\''-m=2 -m=3'\''' -asmflags '' -cppflags '-iquote .' ...`

Reply:
Did you try passing multiple real gc flags and see it gc is OK with it? The command you shared only shows the args Bazel calls `builder compilepkg`, but it doesn't show how `builder` actually calls to gc. Like `compile --first=1 --second=2` is different than `compile '--first=1 --second=2'`.

Reply:
It was wrong initially, after changing to string flag I input "-m=2 -c=1"